shared/components/modals/OfferModal/AddOffer/AddOffer.js

```diff
@@ -92,8 +93,10 @@ export default class AddOffer extends Component {
 
     const { balance, unconfirmedBalance } = currency
 
+    const ethFee = await helpers.eth.estimateFeeValue({ method: 'exchange', speed: 'normal' })
```

Comment:
method: 'send' or 'swap'
```suggestion
    const ethFee = await helpers.eth.estimateFeeValue({ method: 'send', speed: 'normal' })
```